src/apps/concepts/components/ViewConceptsHeader.tsx

```diff
@@ -38,7 +40,8 @@ const ViewConceptsHeader: React.FC<Props> = ({
   containerUrl,
   gimmeAUrl,
   addConceptToDictionary,
-  children
+  children,
+  sources
 }) => {
   const classes = useStyles();
```

Comment:
We need to first format the preferred sources list so that it looks almost like our other sources list.
`const formattedPrefferedSources = Object.entries(PREFERRED_SOURCES_VIEW_ONLY).map(([key, value]) => ({ name: key , url: value }));`